cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/util/ConfigurationFinder.java

```diff
@@ -29,6 +33,44 @@ public class ConfigurationFinder {
   /** Logger instance */
   private static final Logger LOG = LoggerFactory.getLogger(ConfigurationFinder.class);
 
+  /**
+   * load properties from .properties file into TemplateSetConfiguration if found valid properties otherwise load
+   * default values
+   *
+   * @param path to a .properties file
+   * @return TemplateSetConfiguration instance
+   */
+  public static TemplateSetConfiguration loadTemplateSetConfigurations(Path path) {
+
+    Properties props = new Properties();
+    try {
+      props = readConfigurationFile(path);
+    } catch (CobiGenRuntimeException e) {
```

Comment:
I think a more specific Exception like configuration exception would make sense and maybe should be thrown, too. I think the user should get a notification, that the configuration couldn't be read and default values are used.